core/tests/ledger_cleanup.rs

```diff
@@ -233,16 +236,16 @@ mod tests {
     fn test_ledger_cleanup_compaction() {
         solana_logger::setup_with("error,ledger_cleanup::tests=info");
 
-        let ledger_path = get_tmp_ledger_path!();
-        let mut blockstore = Blockstore::open(&ledger_path).unwrap();
+        let blockstore_path = get_tmp_ledger_path!();
+        let mut blockstore = Blockstore::open(&blockstore_path).unwrap();
```

Comment:
I did intentionally switch this variable name and `info!()` prints to be `ledger_path` in my last change to this file. Technically, the blockstore is a subset of the `ledger`, and [the blockstore lives in a sub-directory of `ledger_path`](https://github.com/solana-labs/solana/blob/df6a4930b995b946cf7b2ed68685f3f777dcefcb/ledger/src/blockstore.rs#L361).

Additionally, changing to use `ledger_path` makes the naming consistent with all of the tests in https://github.com/solana-labs/solana/blob/master/ledger/src/blockstore.rs

Reply:
Aghh got it.  I was confused during the merge conflict and didn't realize that.  Will change to ledger!